cases/src/org/commcare/cases/util/CasePurgeFilter.java

```diff
@@ -65,17 +70,16 @@ public CasePurgeFilter(IStorageUtilityIndexed<Case> caseStorage, Vector<String>
     }
 
     private void setIdsToRemoveWithNewExtensions(IStorageUtilityIndexed<Case> caseStorage, Vector<String> owners) {
-        //Create a DAG. The Index will be the case GUID. The Nodes will be a int array containing
-        //[STATUS_FLAGS, storageid], Edges are a string representing the relationship between the
-        //nodes, which is one of the Case Index relationships (IE: parent, extension)
-        DAG<String, int[], String> graph = new DAG<String, int[], String>();
+        // Create a DAG. The Index will be the case GUID. The Nodes will be a int array containing
+        // [STATUS_FLAGS, storageid], Edges are a string representing the relationship between the
+        // nodes, which is one of the Case Index relationships (IE: parent, extension)
+        internalCaseDAG = new DAG<>();
```

Comment:
@phillipm Thanks, I think for some reason at the time I was thinking this is only an issue in javarosa.